### DISCUSSION TITLE
pipeline() global helper

### DISCUSSION THREAD

**Discussion by vanodevium** [laravel/framework#43429] [Ideas]
Like **app()** or **now()**, it would return the new instance of Pipeline.
Also good idea to allow passing _send()_ and _through()_ values.

Was:
```
$posts = app(Pipeline::class)
            ->send(Post::query())
            ->through([
                \App\Filters\Active::class,
                \App\Filters\Sort::class,
            ])
            ->thenReturn()
            ->get();
```
Will be:
```
$posts = pipeline(Post::query(), [
                \App\Filters\Active::class,
                \App\Filters\Sort::class,
            ])
            ->thenReturn()
            ->get();
```

**Comment by henzeb** [comment]
I'd also would like to add an extra optional parameter for the 'via' method.

**Comment by michael-rubel** [comment]
Hi @VanoDevium & @henzeb 

Took this to implement in my package:
https://github.com/michael-rubel/laravel-enhanced-pipeline/pull/3

**Reply by michael-rubel** [reply]
The `pipeline` helper was published in [`v2.1.0`](https://github.com/michael-rubel/laravel-enhanced-pipeline/releases/tag/2.1.0) release of the package.
